.github/workflows/release.yml

```
@@ -0,0 +1,36 @@
+name: Release
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [master, alpha, beta, next]
```

```suggestion
    branches: [main]
```

(comme on n'a pas les autres)